wikimedia/wikipedia-ios
# Notifications - add primary and secondary routing from cells

**Phabricator:**
https://phabricator.wikimedia.org/T288670

### Notes
This PR adds the primary and secondary tap targets and routing in our Notifications Center cells.  It expands on https://github.com/wikimedia/wikipedia-ios/pull/4044 in filling out the `NotificationsCenterCellViewModel` with more data. I am also adding the ability for talk pages to deep link directly to a reply.

### Test Steps
1.. Run on the main/production scheme with an account that has lots of notifications. It's helpful if you're able to find an account with notifications on Commons, Wikidata, and multiple languages. Be sure to add those other languages as an app language in Settings before going to Notification Center. I can share my testing credentials if needed.
2. Browse through notification center via the Explore bell icon. Tap various cells (cell as a whole, and the header link) and see that they are pushed to another screen. In particular find a user talk page cell that has the section header populated. Tap that, and notice that it navigates to a particular reply. There are a lot of types, so we should lean on QA for a more thorough test of that logic.

Google sheet of type > tap destinations: https://docs.google.com/spreadsheets/d/10Gkjd6GWFNDXzPnZhDJ2NvPqNvqoTTCFHMnSUtf3ex8/edit#gid=0
